docs/userguides/alertrules.md

```diff
@@ -0,0 +1,122 @@
+# Associate Users to Alert Rules
+
+Once you [create an alert rule in the Code42 console](https://support.code42.com/Administrator/Cloud/Code42_console_reference/Alert_rule_settings_reference), you can use the CLI add and remove users from your existing alert rules.
+
+Use the `alert-rules` commands to manage which users belong to each rule.
+- To see a list of all the users currently in your organization, you can export a list from the [Users action menu](https://support.code42.com/Administrator/Cloud/Code42_console_reference/Users_reference#Action_menu).
```

Comment:
they could also use the CLI to do this!

Reply:
This is how it was documented in the `detection-lists` user guide so for consistency should I just add a bullet in both sections that mentions you can also do it with the CLI?